backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -34,8 +34,12 @@ If your cloud provider does not support snapshots or if your storage is NFS, you
 
 You create a default `Secret` and then you install the Data Protection Application.
 
-[discrete]
-[role="_additional-resources"]
-== Additional resources
+.Additional resources
 
 * link:https://{velero-domain}/docs/v{velero-version}/locations/[Overview of backup and snapshot locations in the Velero documentation]
+
+include::modules/about-oadp-update-channels.adoc[leveloffset=+1]
```

Comment:
I hope you don't mind a suggestion about the location of this module. I have a feeling that most users skip this assembly and go straight to their installation sections. You might want to consider moving this to the various installation assemblies (installing on AWS, Azure, etc), as a level 2 section, after "Installing the OADP Operator". Alternatively, you could place it before the Operator installation so that the user sees the info before installing the Operator.